NEMO/views/notifications.py

```diff
@@ -84,6 +85,20 @@ def create_buddy_request_notification(buddy_request: BuddyRequest):
             )
 
 
+def create_staff_assistance_request_notification(staff_assistance_request: StaffAssistanceRequest):
+    users: List[User] = User.objects.filter(is_active=True, is_staff=True).exclude(id=staff_assistance_request.user_id)
+    expiration = datetime(staff_assistance_request.creation_time.year + 100, 1, 1)
```

Comment:
let's just use the max
```suggestion
    expiration = datetime.max
```